apps/frontend/src/app/3_organisms/RewardHistory/RewardHistory.tsx

```diff
@@ -0,0 +1,197 @@
+import {
+  applyDataAttr,
+  NotificationType,
+  OrderDirection,
+  OrderOptions,
+  Pagination,
+  Paragraph,
+  ParagraphSize,
+  Table,
+  TransactionId,
+} from '@sovryn/ui';
+
+import { chains, defaultChainId } from '../../../config/chains';
+
+import { ExportCSV } from '../../2_molecules/ExportCSV/ExportCSV';
+import { useNotificationContext } from '../../../contexts/NotificationContext';
+import { useAccount } from '../../../hooks/useAccount';
+import { translations } from '../../../locales/i18n';
+import { EXPORT_RECORD_LIMIT } from '../../../utils/constants';
+import {
+  StabilityDepositChange_Filter,
+  useGetStabilityDepositChangesLazyQuery,
+} from '../../../utils/graphql/zero/generated';
+import { dateFormat } from '../../../utils/helpers';
+import { formatValue } from '../../../utils/math';
+import { useGetRewardHistory } from './hooks/useGetRewardHistory';
+
+const DEFAULT_PAGE_SIZE = 10;
+
+export const RewardHistory: FC = () => {
+  const { t } = useTranslation();
+  const { account } = useAccount();
+  const { addNotification } = useNotificationContext();
+
+  const [page, setPage] = useState(0);
+  const [pageSize, setPageSize] = useState(DEFAULT_PAGE_SIZE);
+  const chain = chains.find(chain => chain.id === defaultChainId);
+
+  const [orderOptions, setOrderOptions] = useState<OrderOptions>({
+    orderBy: 'sequenceNumber',
+    orderDirection: OrderDirection.Desc,
+  });
+
+  const { data, loading } = useGetRewardHistory(
+    account,
+    pageSize,
+    page,
+    orderOptions,
+  );
+  const [getStabilityDeposit] = useGetStabilityDepositChangesLazyQuery();
+
+  const renderCollateralChange = useCallback((collateralGain: string) => {
+    return `${formatValue(
+      Math.abs(Number(collateralGain)),
+      8,
+    )} ${SupportedTokens.rbtc.toUpperCase()}`;
+  }, []);
+
+  const generateRowTitle = useCallback((row: any) => {
+    return (
+      <Paragraph size={ParagraphSize.small}>
+        {dateFormat(row.timestamp)}
+      </Paragraph>
+    );
+  }, []);
+
+  const columns = useMemo(
+    () => [
+      {
+        id: 'sequenceNumber',
+        title: t(translations.rewardHistoryTable.table.timestamp),
+        cellRenderer: (tx: any) => dateFormat(tx.timestamp),
+        sortable: true,
+      },
+      {
+        id: 'stabilityDepositOperation',
+        title: t(translations.rewardHistoryTable.table.transactionType),
+        cellRenderer: row => row.stabilityDepositOperation,
+      },
+      {
+        id: 'collateralGain',
+        title: t(translations.rewardHistoryTable.table['Reward change']),
+        cellRenderer: tx => renderCollateralChange(tx.collateralGain),
+      },
+      {
+        id: 'transactionID',
+        title: t(translations.rewardHistoryTable.table.transactionID),
+        cellRenderer: (tx: any) => (
+          <TransactionId
+            href={`${chain?.blockExplorerUrl}/tx/${tx.hash}`}
+            value={tx.hash}
+            {...applyDataAttr('history-address-id')}
```

Comment:
I actually suggest having all id's on history page with "history" at the start, otherwise your suggestion would make it appear like this is part of reward page content - i.e. `history-reward-address-id`